arm-network/2016-12-01/swagger/applicationGateway.json

```diff
@@ -0,0 +1,1323 @@
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of results."
+        }
+      },
+      "description": "Response for ListApplicationGateways API service call."
+    },
+    "ApplicationGatewayUrlPathMapPropertiesFormat": {
+      "properties": {
+        "defaultBackendAddressPool": {
+          "$ref": "./network.json#/definitions/SubResource",
+          "description": "Default backend address pool resource of URL path map."
+        },
+        "defaultBackendHttpSettings": {
+          "$ref": "./network.json#/definitions/SubResource",
+          "description": "Default backend http settings resource of URL path map."
+        },
+        "pathRules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationGatewayPathRule"
+          },
+          "description": "Path rule of URL path map resource."
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Provisioning state of the backend http settings resource. Possible values are: 'Updating', 'Deleting', and 'Failed'."
+        }
+      },
+      "description": "Properties of UrlPathMap of the application gateway."
+    },
+    "ApplicationGatewayUrlPathMap": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ApplicationGatewayUrlPathMapPropertiesFormat"
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the resource that is unique within a resource group. This name can be used to access the resource."
+        },
+        "etag": {
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "UrlPathMaps give a url path to the backend mapping information for PathBasedRouting."
+    },
+    "ApplicationGatewayWebApplicationFirewallConfiguration": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Defines if the web application firewall is enabled or not."
+        },
+        "firewallMode": {
+          "type": "string",
+          "description": "Web application firewall mode. Possible values are: 'Detection' and 'Prevention'.",
+          "enum": [
+            "Detection",
+            "Prevention"
+          ],
+          "x-ms-enum": {
+            "name": "ApplicationGatewayFirewallMode",
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "enabled"
+      ],
+      "description": "Application gateway web application firewall configuration."
+    },
+    "ApplicationGatewayConnectionDraining": {
+        "properties": {
+            "enabled": {
```

Comment:
Is `enable` required/options? 

If optional, what is the default value for `enabled`?